docs/usage/INVENTORY_SYNC.md

```diff
@@ -129,6 +82,18 @@ __Note__ The statistics object contains the processing time of the last batch on
 
 More examples of how to use the sync [here](/src/integration-test/java/com/commercetools/sync/integration/inventories/InventorySyncIT.java).
 
+##### Usage Tip
+The sync library is not meant to be executed in a parallel fashion. Check the example in [point #2 here](/docs/usage/PRODUCT_SYNC.md#caveats).
```

Comment:
Looks exactly like
https://github.com/commercetools/commercetools-sync-java/pull/341/files#diff-5d9029b724401120cc08d68d8ddc84c4R90

Maybe we can add a file `PARALLELIZATION.md` and refer from everywhere.

Reply:
0532685

e6e5e81